### PR TITLE
Added steps for  scenario

### PR DESCRIPTION
**Pull request by swethan1718**
MOSIP-34512 Minor Resident- A and Adult Resident-B got their UINs and  Resident- B is trying to update Resident- A UIN with   his (Resident-B) biometrics

## Summary by CodeRabbit

* **Tests**
  * Added 30+ end-to-end test scenarios expanding negative and edge-case coverage for registration workflows. New cases cover invalid or malformed inputs, signature/hash validation failures, expired/invalid device credentials, duplicate or missing identity data, and expected rejection/error paths to strengthen system validation of malicious or incorrect submissions.

### CODE REVIEW

File: mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/dsl.properties [hunk 1]
```diff
@@ -11,7 +11,7 @@ scenariosToSkip=
 
 # Add scenarios to include in the the execution list
 # Empty if we want to execute all the scenarios---
-scenariosToExecute=
+scenariosToExecute=216
```
Comment (jayesh12234) [inline mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/dsl.properties]: please remove 216 from above property

File: mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/scenarios.json [hunk 2]
```diff
@@ -25602,6 +25602,527 @@
...
+				"Description": "Updates Demo graphic details and biometric in the persona file",
+				"Input Parameters": "Data to update and persona file path. Other details can be found in the parameter in-line comments",
+				"Return Value": "NA",
+				"Action": "e2e_updateDemoOrBioDetails(iris/*BIO_TYPE*/,0,0/*UPDATE_ATTRIBUTES*/,$$childPersona/*PERSONA_TO_UPDATE*/,$$personaFilePath/*PERSONA_TO_UPDATE_WITH*/)"
+			},
+			"Step-22": {
+				"Description": "Updates persona with UIN",
+				"Input Parameters": "Persona file path and UIN",
+				"Return Value": "NA",
+				"Action": "e2e_updateResidentWithUIN($$personaFilePath1,$$uin2)"
```
Comment (jayesh12234) [inline mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/scenarios.json]: change it to childPersona because he need to use chid persona with above biometric updates

File: mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/scenarios.json [hunk 3]
```diff
@@ -25602,6 +25602,527 @@
...
+			},
+			"Step-23": {
+				"Description": "Generates packet template based on the persona data",
+				"Input Parameters": "Process and persona file path",
+				"Return Value": "Generated Template file path",
+				"Action": "$$updateTemplate=e2e_getPacketTemplate(UPDATE/*PACKET_TYPE*/,$$personaFilePath1)"
+			},
```
Comment (jayesh12234) [inline mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/scenarios.json]: change it to childPersona

File: mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/scenarios.json [hunk 4]
```diff
@@ -25602,6 +25602,527 @@
...
+			"Step-24": {
+				"Description": "Generates and uploads packet with given persona and packet template skipping pre-registration step",
+				"Input Parameters": "Persona file path and template file path",
+				"Return Value": "RID",
+				"Action": "$$rid3=e2e_generateAndUploadPacketSkippingPrereg($$personaFilePath1,$$updateTemplate)"
```
Comment (jayesh12234) [inline mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/scenarios.json]: change it to childPersona

File: mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/scenarios.json [hunk 5]
```diff
@@ -25602,6 +25602,527 @@
...
+				"Description": "Checks the RID status against given packet processing status",
+				"Input Parameters": "Packet processing status and RID",
+				"Return Value": "NA",
+				"Action": "e2e_checkStatus(PROCESSED/*PACKET_STATUS*/,$$rid3)"
+			},
+			"Step-26": {
+				"Description": "Gets the UIN for the given RID",
+				"Input Parameters": "RID",
+				"Return Value": "UIN",
+				"Action": "$$uin3=e2e_getUINByRid($$rid3)"
```
Comment (jayesh12234) [inline mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/scenarios.json]: here as per description uin should not generated then why we are generating uin remove step 26 and 25 add check RID status step and check where it should fail

**Review by jayesh12234 — CHANGES_REQUESTED** [review]

**Review by jayesh12234 — APPROVED** [review]

File: mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/scenarios.json [hunk 6]
```diff
@@ -27054,6 +27054,84 @@
 			"Action": "e2e_DeletePacketData()"
 		}
 	},
+	   "Scenario": "233",
```
[the remark this answers is not in the document]
Comment (swethan1718) [reply mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/scenarios.json]: Fixed, Issue resolved

File: mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/scenarios.json [hunk 7]
```diff
@@ -27054,6 +27054,84 @@
...
+				"Input Parameters": "RID, stage and stage status",
+				"Return Value": "NA",
+				"Action": "e2e_CheckRIDStage($$rid,VALIDATE_PACKET,REJECTED)"
+			},
+			"Step-11": {
+				"Description": "Deletes temporary data generated during packet creation including resident data packet templates and extra files to ensure a clean state for each E2E execution",
+				"Input Parameters": "NA",
+				"Return Value": "NA",
+				"Action": "e2e_DeletePacketData()"
+			}
```
[the remark this answers is not in the document]
Comment (swethan1718) [reply mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/scenarios.json]: Not required for this scenario.
The purpose of Scenario 233 is to validate the rejection workflow rather than the validation of invalid characters in the name attribute.

File: mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/dsl.properties (hunk 1 above)
[the remark this answers is not in the document]
Comment (swethan1718) [reply mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/dsl.properties]: Fixed

File: mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/scenarios.json [hunk 8]
```diff
@@ -25602,6 +25602,527 @@
...
+    "Return Value": "Update response",
+    "Action": "$$emailUpdateResp=e2e_updateResidentEmail($$uin,'resident.updated@email.com')"
+  },
+
+  "Step-14": {
+    "Description": "Validates successful email address update",
+    "Input Parameters": "Email update response",
+    "Return Value": "NA",
+    "Action": "e2e_validateSuccess($$emailUpdateResp)"
+  }
```
[the remark this answers is not in the document]
Comment (swethan1718) [reply mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/scenarios.json]: Issue resolved and fixed

File: mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/scenarios.json [hunk 9]
```diff
@@ -26139,6 +26139,259 @@
...
+			"Input Parameters": "Packet processing status and RID",
+			"Return Value": "NA",
+			"Action": "e2e_checkStatus(PROCESSED/*PACKET_STATUS*/,$$rid2)"
+		},
+		"Step-15": {
+			"Description": "Gets the UIN for the given RID",
+			"Input Parameters": "RID",
+			"Return Value": "UIN",
+			"Action": "$$uin2=e2e_getUINByRid($$rid2)"
+		},
```
[the remark this answers is not in the document]
Comment (swethan1718) [reply mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/scenarios.json]: Not required

File: mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/scenarios.json (hunk 5 above)
[the remark this answers is not in the document]
Comment (swethan1718) [reply mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/scenarios.json]: Fixed

File: mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/scenarios.json (hunk 2 above)
Comment (swethan1718) [reply mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/scenarios.json]: Fixed

File: mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/scenarios.json (hunk 4 above)
Comment (swethan1718) [reply mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/scenarios.json]: Fixed

File: mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/scenarios.json (hunk 5 above)
Comment (swethan1718) [reply mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/scenarios.json]: Removed and modified the steps accordingly

File: mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/scenarios.json (hunk 3 above)
Comment (swethan1718) [reply mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/scenarios.json]: Fixed

File: mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/dsl.properties (hunk 1 above)
Comment (swethan1718) [reply mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/dsl.properties]: Removed

File: mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/scenarios.json [hunk 10]
```diff
@@ -25601,7 +25601,8 @@
 			"Action": "e2e_DeletePacketData()"
 		}
 	},
-	{
+	
+    {
 		"Scenario": "216",
 		"Tag": "Positive_Test",
 		"Persona": "ResidentFemaleAdult",
```
[the remark this answers is not in the document]
Comment (swethan1718) [reply mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/scenarios.json]: Updated the scenario Tag and number as per the instructions

File: mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/scenarios.json [hunk 11]
```diff
@@ -26139,6 +26139,259 @@
 			"Action": "e2e_DeletePacketData()"
 		}
 	},
+	{
+		"Scenario": "220",
+		"Tag": "Postive_Test",
+		"Persona": "ResidentMaleAdult",
+		"Group": "Adult_Update",
+		"Description": "Resident walks into registration center uploads packet and gets UIN. Resident then performs multiple sequential updates name DOB gender and email address.",
```
[the remark this answers is not in the document]
Comment (swethan1718) [reply mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/scenarios.json]: Fixed

**Comment by swethan1718** [comment]
Resolved conflicts

**Comment by swethan1718** [comment]
Resolved conflicts

**Comment by swethan1718** [comment]
Resolved

**Comment by swethan1718** [comment]
Resolved

**Comment by swethan1718** [comment]
Resolved

**Comment by swethan1718** [comment]
Resolved

**Comment by swethan1718** [comment]
Resolved

**Comment by swethan1718** [comment]
resolved

**Comment by swethan1718** [comment]
resolved

**Comment by swethan1718** [comment]
fixed

**Comment by swethan1718** [comment]
fixed

**Comment by swethan1718** [comment]
fixed

**Comment by swethan1718** [comment]
fixed

**Comment by swethan1718** [comment]
fixed

**Comment by swethan1718** [comment]
fixed

**Comment by swethan1718** [comment]
Fixed

File: mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/scenarios.json [hunk 12]
```diff
@@ -25601,6 +25601,97 @@
...
+    "Action": "$$updateResponse=e2e_initiateUINUpdateWithBiometrics($$uinA,$$personaBFilePath)"
+  },
+
+  "Step-11": {
+    "Description": "Validates that the update request fails due to biometric ownership mismatch",
+    "Input Parameters": "Update response",
+    "Return Value": "NA",
+    "Action": "e2e_validateFailure($$updateResponse)"
+  }
+},
```
[the remark this answers is not in the document]
Comment (swethan1718) [reply mosip-acceptance-tests/ivv-orchestrator/src/main/resources/config/scenarios.json]: Fixed

**Review by mohanachandran-s — APPROVED** [review]
